lib/rules/max-top-level-suites.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+/**
+ * @fileoverview Limit the number of top-level suites in a single file
+ * @author Alexander Afanasyev
+ */
+
+var R = require('ramda'),
+    astUtil = require('../util/ast'),
+    defaultSuiteLimit = 1;
+
+module.exports = function (context) {
+    var stack = [],
+        topLevelDescribes = [],
+        suiteLimit = context.options[0];
+
+    if (R.isNil(suiteLimit)) {
+        suiteLimit = defaultSuiteLimit;
+    } else {
+        suiteLimit = suiteLimit.limit;
```

Comment:
It's possible that users supplied an empty object (or that in the future another option will be available and that the user sets only the value of that one).

The way I usually do this is this way:

```js
var options = context.options[0] || {};
var suiteLimit = options.limit || defaultSuiteLimit;
```